pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1166,8 +1167,14 @@ func doActivate(ctx *domainContext, config types.DomainConfig,
 			status.SetErrorNow(err.Error())
 			return
 		}
+		if ds.MountDir != "" {
+			ds.MountDir = path.Join("/mnt/rootfs", ds.MountDir)
+		}
 	}
 
+	// XXX Hard coding mount dir for first disk status
+	status.DiskStatusList[0].MountDir = "/mnt"
```

Comment:
@deitch Here, we are setting up mount directory for the first disk status.

Reply:
Why `/mnt`?

Reply:
https://github.com/lf-edge/eve/blob/f85f18bc69c72a5f2562484fea857dff89d13c63/pkg/xen-tools/initrd/init-initrd#L18

I think we are mounting root in ```/mnt``` in master.